翻译Dashboard和HelloWorld (#1563)
cocos-creator/creator-docs: en/getting-started/helloworld/index.md

# Hello World Project

> Once you understand Cocos Dashboard, let's see how to create and open a Hello World project.

Reviewer: ```suggestion
It's time to create and open the **Hello World** project.
```

> ## Create a new project

Reviewer: ```suggestion
## Creating a new project
```

> In the **Project** tab of Cocos Dashboard, click the **New** button in the bottom right corner to enter the **New Project** page. Select the **empty** project template, set the project name and project path

Reviewer: is it tab or page you wish to you?

Author: The **Project** tab (**项目**选项卡) and the **New Project** page (**新建项目**页面). Nothing wrong with them.

![empty](index/new-empty.png)

Then click the **Create and open** button at the bottom right to automatically create a project with an empty project template and open.

![editor](index/editor-panel.png)

- **Assets**: shows all the assets in the project's assets folder (`assets`).
- **Scene**: working area for displaying and editing the visual content in the scene.
- **Hierarchy**: shows all nodes in the scene and their hierarchical relationships in a tree list, all content seen in the **Scene** panel can be found in the **Hierarchy** panel with corresponding node entries
- **Inspector**: working area for viewing and editing the properties of the currently selected node and its components
- **Assets Preview**: Selecting an asset in the **Assets** panel displays a thumbnail of the asset in the **Assets Preview** panel. If the folder where the asset is located is selected, the thumbnails of all assets under the folder can be displayed for easy viewing.
- **Animation**: used to create some less complex animations that need to be linked with logic, such as UI animations.
- **Console**: used to display error reports, warnings or other log messages generated by the editor and engine.

> For more details about the various editor panels, please refer to [Editor Panels](../../editor/index.md).

Reviewer: ```suggestion
For more information about the various editor panels, please refer to [Editor Panels](../../editor/index.md).
```

Reviewer: is it tab or page you wish to you?

      // dummy = '';

      /* use `property` decorator if you want the member to be serializable */
      // @property
      // serializableDummy = 0;

      start () {
          // Your initialization goes here.
          console.info('Hello world');
      }

      // update (deltaTime: number) {
      // // Your update function goes here.
      // }
  }
  ```

- Binding Scripts

  Select the created Cube node in the **Hierarchy** panel and click **Add Component -> Custom Script -> HelloWorld** at the bottom of the **Inspector** panel to mount the script to the Cube node. Alternatively, you can drag and drop the script directly into the **Inspector** panel.

  ![script](index/script.png)

  > **Note**: remember to save the scene after setting it up.

## Previewing the Scene

Once the simple scene setup is done, you can click the **Play on Device** button on top of the editor to preview the game. Currently, previewing is supported using **Browser/Simulator/GameView**.

![preview](index/preview.png)

Using the **Browser** preview as an example, Cocos Creator will run the current game scene with your default browser, as shown in the following image.

![console](index/console.png)

> You can see the output "Hello World" in the log message in the developer tools of the browser.

Reviewer: ```suggestion
Notice the output "Hello World" in the log message in the developer tools of the browser.
```
